src/yaml/builder.cr

```diff
@@ -130,11 +130,45 @@ class YAML::Builder
     yield.tap { end_mapping }
   end
 
-  def alias(anchor : String)
+  # Emits an alias to the given *anchor*.
+  #
+  # ```crystal
+  # require "yaml"
+  #
+  # yaml = YAML.build do |builder|
+  #   builder.mapping do
+  #     builder.scalar "key"
+  #     builder.alias "example"
+  #   end
+  # end
+  #
+  # yaml # => "---\nkey: *example\n"
+  # ```
+  def alias(anchor : String) : Nil
     LibYAML.yaml_alias_event_initialize(pointerof(@event), anchor)
     yaml_emit("alias")
   end
 
+  # Emits the scalar "<<" followed by an alias to the given *anchor*.
+  #
+  # See [YAML Merge](https://yaml.org/type/merge.html).
+  #
+  # ```crystal
+  # require "yaml"
+  #
+  # yaml = YAML.build do |builder|
+  #   builder.mapping do
+  #     builder.merge "development"
+  #   end
+  # end
+  #
+  # yaml # => "---\n<<: *development\n"
+  # ```
+  def merge(anchor : String) : Nil
+    self.scalar "<<"
```

Comment:
```suggestion
   scalar "<<"
```

Reply:
Used self just so its consistent with the alias call below.

Reply:
It's redundant in both places, so why not omit it?

Reply:
🤷‍♂ Consistency sake?

Reply:
> EDIT: It's required for the `alias` method call for the same reason it needs `itself`.

Calling methods without a receiver implicitly uses `self`, so what would be the reason to do so?

Reply:
![image](https://user-images.githubusercontent.com/12136995/60626133-ce771f00-9db7-11e9-847d-0a7826a02df7.png)

Reply:
![image](https://user-images.githubusercontent.com/12136995/60626159-ecdd1a80-9db7-11e9-9071-cef6b0f18472.png)

Reply:
My bad then, forgot it's a language keyword 🤦‍♂